### ISSUE TITLE
Add Data Subset in "Series Spectra" plot

### ISSUE THREAD

**Issue by borondics** [Quasars/orange-spectroscopy#380]
It would be necessary to be able to transmit and visualize selections in all the spectral plots, in this particular case the "Series Spectra" so that the user can see selections from another (e.g. PCA) space projected onto the spectra.

**Comment by stuart-cls** [comment]
I agree but I don't think I'll be able to work on this in the near future.

**Comment by borondics** [comment]
I'll try to find time for this one. There are many others we should do to this widget though! I know that SIRIUS is interested in this, so I'll ask them too.
